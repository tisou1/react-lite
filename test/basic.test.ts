import { describe, it, expect } from 'vitest'


describe('basic', () => {
  it('add', () => {
    expect(1 + 1).toEqual(2)
  })
})