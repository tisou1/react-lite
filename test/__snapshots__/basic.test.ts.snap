// Vitest Snapshot v1

exports[`basic > add 1`] = `2`;
